### PR TITLE
Embedded flow

### PR DESCRIPTION
**Pull request by n0rdy**

### CODE REVIEW

File: src/main/resources/templates/psu_data.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <h6 class="form-group">Transactions period</h6>
+        <div class="form-group">
+            <label for="dateFrom">Date from</label>
+            <input type="date" class="form-control" id="dateFrom">
+        </div>
+        <div class="form-group">
+            <label for="dateTo">Date to</label>
+            <input type="date" class="form-control" id="dateTo">
+        </div>
+        <button type="button" class="btn btn-primary" id="addProjectBtn" onclick="submitPsuData();">Створити</button>
```
Comment (berezkin88) [inline src/main/resources/templates/psu_data.html]: I have doubts that Germans know Ukrainian)

Comment (n0rdy) [reply src/main/resources/templates/psu_data.html]: Sad story =)
Fixed

File: src/main/java/de/adorsys/xs2a/adapter/ui/paging/PageController.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@Controller
+public class PageController {
+
+    @GetMapping()
+    public String intro() {
+        return "intro";
+    }
+
+    @GetMapping("/page/psu/data")
```
Comment (spetrychenko) [inline src/main/java/de/adorsys/xs2a/adapter/ui/paging/PageController.java]: not sure that we need three levels of URL nesting, I propose to continue with `/page/psu-data`. 
what do you think?

Comment (n0rdy) [reply src/main/java/de/adorsys/xs2a/adapter/ui/paging/PageController.java]: done, thanks

**Pull request merged**